### DISCUSSION TITLE
Hi, I updated the Japanese translation file today.

### DISCUSSION THREAD

**Discussion by KKbion** [Klocman/Bulk-Crap-Uninstaller#507] [General]
Hi, I updated the Japanese translation file today.

[2023jun19_JP_BCULocalisation_Pack.zip](https://github.com/Klocman/Bulk-Crap-Uninstaller/files/11786121/2023jun19_JP_BCULocalisation_Pack.zip)

As you pointed out the other day, I have changed the notation as "アンインストール lists|.bcul" -> "アンインストールリスト|.bcul".

Also, since the contents of Silent Uninstall and Quiet Uninstall seemed to be almost the same, I have unified the notation to "Silent Uninstall" for both.

**Comment by Klocman** [comment]
Hello, and than you for the update! The translation seems good to me, here's a debug build if you want to test it [net6.0-windows10.0.18362.0.zip](https://github.com/Klocman/Bulk-Crap-Uninstaller/files/11951599/net6.0-windows10.0.18362.0.zip)

**Reply by KKbion** [reply]
Thank you for your confirmation!

**Comment by Klocman** [comment]
109ae7a9fd6b4bc6660c93b23faf0220a99b7f52
